frontend/code-viewer.ts

```diff
@@ -145,6 +145,14 @@ export class CodeViewer extends LitElement {
       .replace(/'/g, "&#039;");
   }
 
+  removeMetaInfoFromCode(code: string) {
+    return code
+      .substring(code.indexOf("import"))
+      .replace(code.substring(code.indexOf("@Metadata"), code.indexOf("public class")), "")
```

Comment:
Does this really work or are there some recipies with annotations after `@Metadata`?

Reply:
There are actually ones with `@JsModule`

Reply:
Like this one: https://cookbook.vaadin.com/copy-to-clipboard

Reply:
Improved the sanitization logic in a new commit